FND/OwnershipAndControl/Ownership.rdf

```diff
@@ -126,13 +126,13 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-oac-own;hasOwnedAsset"/>
-				<owl:someValuesFrom rdf:resource="&fibo-fnd-pty-pty;Undergoer"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-oac-own;Asset"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-oac-own;hasOwningParty"/>
```

Comment:
the name owningParty is misleading since the property is for the role not the party

Reply:
> the name owningParty is misleading since the property is for the role not the party

Do you have a suggestion for an alternative?